packages/jest-types/src/Config.ts

```diff
@@ -330,7 +330,7 @@ export type ProjectConfig = {
   dependencyExtractor?: string;
   detectLeaks: boolean;
   detectOpenHandles: boolean;
-  displayName?: DisplayName;
+  displayName?: DisplayName | string;
```

Comment:
this is incorrect - by the time it's `ProjectConfig` it's been normalized into an object

Reply:
Even when specifying in a `jest.config.ts` file?

My config file looks something like...

```ts
import type { Config } from '@jest/types';

const config: Config.InitialOptions = {
    projects: [
        {
            displayName: 'hello', // <-- this is a type error for `ProjectConfig` (expects a `DisplayName`)
            moduleNameMapper: { // <-- this is a type error for `ProjectConfig` (expects string[])
                '\\.(css|scss)$': '<rootDir>/__mocks__/styleMock.js'
            }
        }
    ]
}
```

Should there be a different type for specifying `projects` in `Config.InitialOptions`?

Reply:
yep, if you ignore the type error it should work correctly.

Not sure the best way to type `projects` in `InitialOptions`. It's probably a `Pick` of the project config from `InitialOptions`, but that sounds icky...

Reply:
we should probably have a bunch more tests for `projects` normalization in https://github.com/facebook/jest/blob/master/packages/jest-config/src/__tests__/normalize.test.ts, but I'm 99% sure I'm correct on it working correctly 😅

Reply:
It almost seems like the type should be...

```ts
projects: Array<Glob> | Array<Omit<InitialOptions, 'projects'>>;
```

Reply:
yep, essentially. there's a bunch of `GlobalConfig` options there as well that should be excluded as well, tho. Can you do like `keyof GlobalConfig` and exclude those?

Reply:
Yup, that works!